### PR TITLE
Add more checks

### PR DESCRIPTION
**Pull request by Jason2605**
#### Changes proposed in this pull request: 
- Cars were still being checked on the license no matter the side, extra checks have been added to make sure side is not an issue.

### CODE REVIEW

**Comment by BoGuu** [comment]
We should add all the Apex stuff into the normal configs now I think (copy paste them in, w/e). Up for it :) ?

**Comment by Jason2605** [comment]
Alright, when you merge that one in, ill redo this.

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 1]
```diff
@@ -578,60 +623,60 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "\a3\air_f\Heli_Light_01\Data\Skins\heli_light_01_ext_digital_co.paa"
-            } }
+            }, "" }
         };
     };
 
     class B_Heli_Light_01_F {
         vItemSpace = 90;
-        conditions = "license_civ_pilot || {license_cop_cAir} || {license_med_mAir}";
+        conditions = "license_civ_pilot || {license_cop_cg} || {license_civ_mAir}";
```
Comment (Jawshy) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: Police need a coast guard license in order to buy the Hummingbird. _Appears to be legitimate._ :kappa:

Comment (Jawshy) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: `license_med_mAir` was [correct](https://github.com/AsYetUntitled/Framework/blob/33213795c08f082fecd048ced786f06e3580f3c4/Altis_Life.Altis/config/Config_SpyGlass.hpp#L24).

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 2]
```diff
@@ -642,68 +687,68 @@ will modify the virtual space and the price of the vehicle, but other informatio
 
     class O_Heli_Light_02_unarmed_F {
         vItemSpace = 210;
-        conditions = "license_civ_pilot || {license_med_mAir}";
+        conditions = "license_civ_pilot || {license_civ_mAir} || !(playerSide isEqualTo civilian)";
```
Comment (Jawshy) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: `license_med_mAir` was [correct](https://github.com/AsYetUntitled/Framework/blob/33213795c08f082fecd048ced786f06e3580f3c4/Altis_Life.Altis/config/Config_SpyGlass.hpp#L24).

File: Altis_Life.Altis/config/Config_Vehicles.hpp (hunk 1 above)
Comment (Jason2605) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: cAir  != coastguard. And the rest I'll change, must have been my copy paste.

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 3]
```diff
@@ -198,14 +243,14 @@ class LifeCfgVehicles {
 
     class C_Rubberboat {
         vItemSpace = 45;
-        conditions = "license_civ_boat";
+        conditions = "license_civ_boat || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: || {!(playerSide isEqualTo civilian)}

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 4]
```diff
@@ -198,14 +243,14 @@ class LifeCfgVehicles {
...
         price = 5000;
         textures[] = { };
     };
 
     class B_Heli_Transport_01_F {
         vItemSpace = 200;
-        conditions = "license_cop_cAir";
+        conditions = "license_cop_cAir || !(playerSide isEqualTo west)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: || {!(playerSide isEqualTo west)}

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 5]
```diff
@@ -219,34 +264,34 @@ class LifeCfgVehicles {
...
                 "#(argb,8,8,3)color(0.05,0.05,0.05,1)"
-            } }
+            }, "" }
         };
     };
 
     class B_Boat_Armed_01_minigun_F {
         vItemSpace = 175;
-        conditions = "license_cop_cg";
+        conditions = "license_cop_cg || !(playerSide isEqualTo west)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo west)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 6]
```diff
@@ -219,34 +264,34 @@ class LifeCfgVehicles {
...
         price = 75000;
         textures[] = { };
     };
 
     class B_Boat_Transport_01_F {
         vItemSpace = 45;
-        conditions = "license_cop_cg";
+        conditions = "license_cop_cg || !(playerSide isEqualTo west)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo west)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 7]
```diff
@@ -219,34 +264,34 @@ class LifeCfgVehicles {
...
         price = 3000;
         textures[] = { };
     };
 
     class O_Truck_03_transport_F {
         vItemSpace = 285;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 8]
```diff
@@ -219,34 +264,34 @@ class LifeCfgVehicles {
...
         price = 200000;
         textures[] = { };
     };
 
     class O_Truck_03_device_F {
         vItemSpace = 350;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 9]
```diff
@@ -281,87 +326,87 @@ class LifeCfgVehicles {
 
     class B_G_Offroad_01_armed_F {
         vItemSpace = 65;
-        conditions = "license_civ_rebel";
+        conditions = "license_civ_rebel || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 10]
```diff
@@ -281,87 +326,87 @@ class LifeCfgVehicles {
...
         price = 750000;
         textures[] = { };
     };
 
     class C_Boat_Civil_01_F {
         vItemSpace = 85;
-        conditions = "license_civ_boat";
+        conditions = "license_civ_boat || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 11]
```diff
@@ -281,87 +326,87 @@ class LifeCfgVehicles {
...
-        conditions = "license_cop_cg";
+        conditions = "license_cop_cg || !(playerSide isEqualTo west)";
         price = 20000;
         textures[] = { };
     };
 
     class B_Truck_01_box_F {
         vItemSpace = 450;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 12]
```diff
@@ -281,87 +326,87 @@ class LifeCfgVehicles {
...
         price = 275000;
         textures[] = { };
     };
 
     class O_MRAP_02_F {
         vItemSpace = 60;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 13]
```diff
@@ -281,87 +326,87 @@ class LifeCfgVehicles {
...
         price = 150000;
         textures[] = { };
     };
 
     class C_Offroad_01_F {
         vItemSpace = 65;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 14]
```diff
@@ -281,87 +326,87 @@ class LifeCfgVehicles {
...
                 "#(ai,64,64,1)Fresnel(1.3,7)"
-            } }
+            }, "" }
         };
     };
 
     class C_Kart_01_Blu_F {
         vItemSpace = 20;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 15]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
 
     class C_Hatchback_01_sport_F {
         vItemSpace = 45;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 16]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "#(ai,64,64,1)Fresnel(1.3,7)"
-            } }
+            }, "" }
         };
     };
 
     class B_Quadbike_01_F {
         vItemSpace = 25;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 17]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "#(argb,8,8,3)color(0.05,0.05,0.05,1)"
-            } }
+            }, "" }
         };
     };
 
     class I_Truck_02_transport_F {
         vItemSpace = 200;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 18]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "#(argb,8,8,3)color(0.05,0.05,0.05,1)"
-            } }
+            }, "" }
         };
     };
 
     class O_Truck_03_covered_F {
         vItemSpace = 300;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 19]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
         price = 250000;
         textures[] = {};
     };
 
     class C_Hatchback_01_F {
         vItemSpace = 40;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 20]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "\a3\soft_f_gamma\Hatchback_01\data\hatchback_01_ext_base09_co.paa"
-            } }
+            }, "" }
         };
     };
 
     class C_SUV_01_F {
         vItemSpace = 50;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 21]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "\a3\soft_f_gamma\SUV_01\Data\suv_01_ext_02_co.paa"
-            } }
+            }, "" }
         };
     };
 
     class C_Van_01_transport_F {
         vItemSpace = 100;
-        conditions = "license_civ_driver";
+        conditions = "license_civ_driver || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 22]
```diff
@@ -380,182 +425,182 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
                 "\a3\soft_f_gamma\Van_01\Data\van_01_ext_red_co.paa"
-            } }
+            }, "" }
         };
     };
 
     class C_Van_01_box_F {
         vItemSpace = 150;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 23]
```diff
@@ -642,68 +687,68 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
+        conditions = "license_civ_boat || {license_cop_cg} || (playerSide isEqualTo independent)";
         price = 150000;
         textures[] = {};
     };
 
         class C_Van_01_fuel_F {
         vItemSpace = 20;
         vFuelSpace = 19500;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 24]
```diff
@@ -642,68 +687,68 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
-            } }
+            }, "" }
         };
     };
 
     class I_Truck_02_fuel_F {
         vItemSpace = 40;
         vFuelSpace = 42000;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 25]
```diff
@@ -642,68 +687,68 @@ will modify the virtual space and the price of the vehicle, but other informatio
...
-            } }
+            }, "" }
         };
     };
 
     class B_Truck_01_fuel_F {
         vItemSpace = 50;
         vFuelSpace = 50000;
-        conditions = "license_civ_trucking";
+        conditions = "license_civ_trucking || !(playerSide isEqualTo civilian)";
```
Comment (WolfSkin1337) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: ` || {!(playerSide isEqualTo civilian)}`

**Comment by WolfSkin1337** [comment]
huch sry.
for this massiv lines

**Comment by SimZor** [comment]
You need a lazy eval, didn't work without it for me.

**Comment by WolfSkin1337** [comment]
@Jason2605 what are with this?

**Pull request closed without merging**